### PR TITLE
Archivematica 1.7 packaging

### PR DESCRIPTION
**Pull request by scollazo**
This PR is built on top of #46, and is focused on changes to archivematica packages.

 I'm creating a different PR to make CR easier, and once #46 is merged, I'll rebase this one.

### CODE REVIEW

**Comment by sevein** [comment]
@scollazo, before CRing this or merging I'm wondering if we should tag 8ae1bfd (master) as v1.6.1? I was thinking that would be enough if we ever want to be able to generate those packages in the future right? Would that be a good idea?

File: debs/siegfried/Makefile [hunk 1]
```diff
@@ -0,0 +1,60 @@
+NAME          = siegfried
+PACKAGE       = siegfried
+VERSION       ?= 1.7.3
```
Comment (sevein) [inline debs/siegfried/Makefile]: AM17 is going to use siegfried 1.7.6. There is already a PR in place: https://github.com/artefactual/archivematica-fpr-admin/pull/65.

File: rpm/six/Makefile [hunk 2]
```diff
@@ -0,0 +1,43 @@
+NAME          = six
```
Comment (sevein) [inline rpm/six/Makefile]: Can you remind me why are we building six? This is probably a dependency in one of our AM components but that's managed via pip in the virtualenvs. Is this used by someone else?

File: rpm/olefile/Makefile [hunk 3]
```diff
@@ -0,0 +1,43 @@
+NAME          = olefile
```
Comment (sevein) [inline rpm/olefile/Makefile]: What is `olefile` used for? It seems unused?

File: rpm/archivematica/etc/archivematica-storage-service.service [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+After=network.target
+
+[Service]
+
+PIDFile=/run/archivematica-storage-service_gunicorn.pid
+User=archivematica
+Group=archivematica
+EnvironmentFile=-/etc/sysconfig/archivematica-storage-service
+WorkingDirectory=/usr/share/archivematica/storage-service/
+ExecStart=/usr/lib/python2.7/archivematica/storage-service/bin/gunicorn --workers 2 --timeout 120 --access-logfile /var/log/archivematica/storage-service/gunicorn.log --error-logfile /var/log/archivematica/storage-service/gunicorn_error.log --log-level error --bind localhost:7500 storage_service.wsgi:application
```
Comment (sevein) [inline rpm/archivematica/etc/archivematica-storage-service.service]: Should we use `/etc/archivematica/storage-service.gunicorn-config.py`?

**Comment by sevein** [comment]
On a side note, I think that https://github.com/artefactual/archivematica/issues/776 can be considered a blocker. I'm going to be working on this today - it will allow us to include a logging configuration file in our 1.7 packages to mimic the old configuration used in 1.6.

**Comment by scollazo** [comment]
Done, I didn't knew you can make that,  thanks for the tip!

**Comment by sevein** [comment]
> Done, I didn't knew you can make that, thanks for the tip!

It gets a bit messy if you want to make a change in the target (dev/package-updates) because it will make you rebase this branch after (dev/archivematica-1.7) otherwise the PR becomes a mess.

File: rpm/archivematica/etc/archivematica-storage-service.service (hunk 4 above)
Comment (scollazo) [reply rpm/archivematica/etc/archivematica-storage-service.service]: Done

File: debs/trusty/archivematica/debian-MCPClient/postinst [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+userID=`id -u archivematica`
+
+if [ "${userID}" = 333 ]; then
+  echo "User archivematica exists"
+else
+  adduser --uid 333 --group --system --home /var/lib/archivematica/ archivematica
+fi
+
+# use pip installed fido instead of os package version
+update-alternatives --install /usr/bin/fido fido /usr/local/bin/fido 1
```
Comment (sevein) [inline debs/trusty/archivematica/debian-MCPClient/postinst]: Do we still need this?

**Comment by scollazo** [comment]
Current master is more close to 1.4.0, than to 1.6. Both , 1.5 and 1.6 changes never arrived to master.

**Pull request merged**